licenses/check_licenses.dart

```diff
@@ -0,0 +1,142 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:core' hide print;
+import 'dart:io' hide exit;
+import 'package:meta/meta.dart';
+import 'package:path/path.dart' as path;
+
+import 'run_command.dart';
+import 'utils.dart';
```

Comment:
i recommend inlining these files into this one, and removing as much of the code as possible, e.g. it seems runCommand isn't needed at least

Reply:
Done, moved all the functions to check_licenses.dat and removed the utility files.